xl2times/datatypes.py

```diff
@@ -156,13 +156,16 @@ class Config:
     veda_attr_defaults: Dict[str, Dict[str, list]]
     # Known columns for each tag
     known_columns: Dict[Tag, Set[str]]
+    # Names of regions to include in the model; if empty, all regions are included.
+    filter_regions: Iterable[str]
```

Comment:
Btw, we probably want to say `List[str]` and not `Iterable`. The latter also includes generators, which you can only iterate through once (the second time you go through them it'll be empty). But in the code below you check if something is in `filter_regions` multiple times.